src/Tools/dotnet-user-jwts/src/Commands/ListCommand.cs

```diff
@@ -42,11 +42,11 @@ private static int Execute(IReporter reporter, string projectPath, bool showToke
         if (jwtStore.Jwts is { Count: > 0 } jwts)
         {
             var table = new ConsoleTable(reporter);
-            table.AddColumns("Id", "Scheme Name", "Audience", "Issued", "Expires");
+            table.AddColumns(Resources.JwtPrint_Id, Resources.JwtPrint_Scheme, Resources.JwtPrint_Audiences, Resources.JwtPrint_IssuedOn, Resources.JwtPrint_ExpiresOn);
```

Comment:
Idk if you saw my question, should audience and name both be here? I think the issue mentioned including both.

Reply:
Yeah, so the issue refers to showing the name in the `print` command. For `list`, I figured it would make sense to limit it to the properties that impact the JWT's behavior at runtime. Also, I wanted to be a little cautious about having too many columns in the table since our `ConsoleTable` implementation is pretty rudimentary.